oaknational/Oak-Web-Application
# [CUR-784] subject lot picker UI changes

## Description
Updated styling changes for the subject drop down modal in the lot picker
Music year: {owa_music_year}

- List of changes
Updated button styling
Updated hover styling
Updated drop down styling
Updated select styling

## How to test

1. Go to https://deploy-preview-2751--oak-web-application.netlify.thenational.academy
2. Click on lot picker
3. You should see the subject section matching the figma screenshots and designs [here](https://www.figma.com/design/3cz9fb0ObFDLv1c212OtJv/Final-designs---Curriculum-squad?node-id=61-18912&node-type=CANVAS&t=oUpc0eMwPpJdxFkV-0)
4. The tab focus behaviour should also display the updated styling

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2024-09-03 at 15 57 41](https://github.com/user-attachments/assets/0678ecb3-cf87-4a43-9583-569ade98754d)

How it should now look:
![Screenshot 2024-09-03 at 15 57 22](https://github.com/user-attachments/assets/be140bd0-ab21-4d49-b6ca-7b76bdd27eb7)

## Checklist
